tasks/storage.py

```diff
@@ -16,4 +16,4 @@ def mark_task_completed(index: int, completed: bool) -> None:
 
 
 def get_all_tasks() -> List[Tuple[int, str, bool]]:
-    return [(i, task["title"], task["completed"]) for i, task in enumerate(_DB)]
+    return [(i + 1, task["title"], task["completed"]) for i, task in enumerate(_DB)]
```

Comment:
Good, but you didn't change two other methods - delete and mark done. When user input 1 you should delete or mark done first item, and it has index 0, not 1